apps/test/unit/templates/studioHomepages/TeacherSectionsTest.jsx

```diff
@@ -4,7 +4,8 @@ import {expect} from '../../../util/configuredChai';
 import {
   UnconnectedTeacherSections as TeacherSections
 } from '@cdo/apps/templates/studioHomepages/TeacherSections';
-import ContentContainer from "@cdo/apps/templates/ContentContainer";
+import { UnconnectedContentContainer  as ContentContainer
+} from "@cdo/apps/templates/ContentContainer";
```

Comment:
Whitespace turned out funny here. Prefer the pattern used with `UnconnectedTeacherSections` above if this line is too long.